src/mangoapp.cpp

```diff
@@ -0,0 +1,21 @@
+#include <sys/ipc.h>
+#include <sys/shm.h>
+
+#include "steamcompmgr.hpp"
+
+static bool inited = false;
+static char *shm_str = "";
+
+void init_mangoapp(){
+    key_t key = ftok("mangoapp",65);
+    int shmid = shmget(key,1024,0666|IPC_CREAT);
+    shm_str = (char*) shmat(shmid,(void*)0,0);
+    inited = true;
+}
+
+void frame_timing(){
+    if (!inited)
+        init_mangoapp();
+
+    snprintf(shm_str, 1, "%s", "");
```

Comment:
I guess I am a bit confused why you are using snprintf to print an empty string via a "%s"...

Reply:
I've reworked this a bit to use a struct over shm, might look less stupid now